Doc/library/decimal.rst

```diff
@@ -23,6 +23,11 @@
    # make sure each group gets a fresh context
    setcontext(Context())
 
+.. testcleanup:: *
+
+   # make sure other tests (also outside this file) get a fresh context
+   setcontext(Context())
+
```

Comment:
The `setcontext(Context())` in `testsetup` does not guarantee that the tests outside this file get a fresh context.  We need to clean the context at the end of the tests.